server/plugin/plugin_test.go

```diff
@@ -51,12 +51,26 @@ func TestMessageWillBePosted(t *testing.T) {
 			isValidLink: true,
 		},
 		{
-			description: "MessageWillBePosted: test change post for valid link",
+			description: "MessageWillBePosted: test change post for valid pull request link",
 			data:        []string{"https:", "", "dev.azure.com", "abc", "xyz", "_git", "xyz", "pullrequest", "1"},
 			message:     "mockMessage",
 			isValidLink: true,
 			link:        "https://dev.azure.com/abc/xyz/_git/xyz/pullrequest/1",
 		},
+		{
+			description: "MessageWillBePosted: test change post for valid build pipeline link",
+			data:        []string{"https:", "", "dev.azure.com", "abc", "xyz", "_build", "results?buildId=50&view=results"},
```

Comment:
We avoid using the actual value. 
```suggestion
			data:        []string{"https:", "", "test.com", "abc", "xyz", "_build", "results?buildId=50&view=results"},
```